core/experiment-2/packages/admin/next.config.js

```diff
@@ -0,0 +1,27 @@
+const NextFederationPlugin = require('@module-federation/nextjs-mf');
+// this enables you to use import() and the webpack parser
+// loading remotes on demand, not ideal for SSR
+const remotes = isServer => {
+  const location = isServer ? 'ssr' : 'chunks';
+  return {
+    plugins: `plugins@http://localhost:3001/_next/static/${location}/remoteEntry.js`,
```

Comment:
Let me see if I understood. Are you suggesting we keep the plugins repo at a single endpoint so that WebPack module knows it at build time without interfering in future updates of the plugins themselves? That's smart.

Reply:
This may have a disadvantage because I don't know where my plug-in is introduced. 
tree shaking may not work